copier/main.py

```diff
@@ -847,11 +851,12 @@ def _apply_update(self):
             self._execute_tasks(
                 self.template.migration_tasks("before", self.subproject.template)
             )
-            # Clear last answers cache to load possible answers migration
-            with suppress(AttributeError):
-                self.answers = AnswersMap()
-            with suppress(AttributeError):
-                del self.subproject.last_answers
+            # Clear last answers cache to load possible answers migration, if skip_answered flag is not set
+            if self.skip_answered is False:
+                with suppress(AttributeError):
+                    self.answers = AnswersMap()
```

Comment:
This suppress looks like unnecessary, isn't it?

```suggestion
                self.answers = AnswersMap()
```